acestep/training/dataset_builder_modules/preprocess_vae.py

```
@@ -1,12 +1,150 @@
+    vae_device = next(vae.parameters()).device
+    vae_dtype = vae.dtype
+
+    # Auto-select chunk size based on GPU VRAM
+    if chunk_size is None:
+        gpu_mem_gb = 0.0
+        if torch.cuda.is_available():
+            try:
+                props = torch.cuda.get_device_properties(vae_device)
+                gpu_mem_gb = props.total_mem / (1024 ** 3)
+            except Exception:
+                pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)